src/chigame/users/urls.py

```diff
@@ -31,8 +32,10 @@
     path("add_friend/<int:pk>", view=send_friend_invitation, name="add-friend"),
     path("remove_friend/<int:pk>", view=remove_friend, name="remove-friend"),
     path("cancel_friend_invitation/<int:pk>", view=cancel_friend_invitation, name="cancel-friend-invitation"),
+    path("invite_to_game/<int:pk>", view=invite_to_game, name="invite-to-game"),
```

Comment:
I don't think these urls are correct because they don't have the pk for the match which is needed for the view